packages/help-center/src/components/utils.tsx

```diff
@@ -123,3 +127,47 @@ export const matchSupportInteractionId = (
 		return foundMatch;
 	}
 };
+
+export const useGetMostRecentOpenConversation = () => {
+	const { isChatLoaded } = useSelect( ( select ) => {
+		const store = select( HELP_CENTER_STORE ) as HelpCenterSelect;
+		return {
+			isChatLoaded: store.getIsChatLoaded(),
+		};
+	}, [] );
+
+	const { data: supportInteractionsResolved, isLoading: isLoadingResolvedInteractions } =
+		useGetSupportInteractions( 'zendesk', 100, 'resolved' );
+
+	const { data: supportInteractionsOpen, isLoading: isLoadingOpenInteractions } =
+		useGetSupportInteractions( 'zendesk', 10, 'open' );
+
+	const isLoadingInteractions = isLoadingResolvedInteractions || isLoadingOpenInteractions;
+
+	if ( isChatLoaded && getZendeskConversations && ! isLoadingInteractions ) {
+		const allConversations = getZendeskConversations();
+		const supportInteractions = [
+			...( supportInteractionsResolved || [] ),
+			...( supportInteractionsOpen || [] ),
+		];
+
+		const filteredConversations = getConversationsFromSupportInteractions(
+			allConversations,
+			supportInteractions
+		);
+
+		const recentConversations = filteredConversations?.sort( ( conversationA, conversationB ) => {
+			const aCreatedAt = conversationA.metadata?.createdAt;
+			const bCreatedAt = conversationB.metadata?.createdAt;
+			if ( aCreatedAt && bCreatedAt ) {
+				return new Date( bCreatedAt ).getTime() - new Date( aCreatedAt ).getTime();
+			}
+			return 0;
+		} );
```

Comment:
Can we extract the method to sort an array to its own?
Also probably better to change the name to `sortedConversations`, as `recent` may lead to think we're just grabbing the most recent conversations.